pkg/cluster/internal/create/actions/kubeadminit/init.go

```diff
@@ -35,6 +30,8 @@ import (
 	"sigs.k8s.io/kind/pkg/cluster/internal/loadbalancer"
 	"sigs.k8s.io/kind/pkg/cluster/nodes"
 	"sigs.k8s.io/kind/pkg/exec"
+
+	"k8s.io/client-go/tools/clientcmd"
```

Comment:
nit: external imports should exist in the 2nd block (after stdlib imports but before imports within this module)